src/jarabe/journal/journalactivity.py

```diff
@@ -479,6 +651,12 @@ def freeze_ui(self):
     def unfreeze_ui(self):
         self._set_widgets_sensitive_state(True)
 
+    def initialize_journal_object(self, title=None, bundle_id=None, 
```

Comment:
what is the point of this method?  Can't people just call initialize_journal_object?